### PR TITLE
Update new UI screenshot and add instruction for using token in swagger

### PR DESCRIPTION
**Pull request by yingqi-chen**

### CODE REVIEW

File: HOW-TO.md [hunk 1]
```diff
@@ -3,10 +3,10 @@ You must have an active CyberGRX account that has the ability to manage users.
 
 The workflow is pretty simple:
  - Enter `Manage my company user accounts` using the top right navigational element.
- - Click the tab to `Manage Access Tokens`
+ - Click the tab to `Access Tokens`
  - Click the button to `Add a new token`
  - Accept the promt creating a token.
- - Show the secret and copy it, this is the only time that this token will be available over the API!
+ - Click `view the secret` the secret and copy it, this is the only time that this token will be available over the API!
```
Comment (miriamgrigsby) [inline HOW-TO.md]: Click `view the secret` to view the secret and then copy it, this is the only time that this token will be available over the API!

**Review by miriamgrigsby — CHANGES_REQUESTED** [review]
quick grammar fix then looks good to me!

File: HOW-TO.md (hunk 1 above)
Comment (yingqi-chen) [reply HOW-TO.md]: @miriamgrigsby Fixed!

**Comment by yingqi-chen** [comment]
Remove useless msgs

**Review by timthesinner — APPROVED** [review]

**Pull request merged**